dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -914,51 +941,120 @@ static void constructPartname(const char *filename,unsigned n, StringBuffer &pn,
     }
 }
 
-static bool parseFileName(const char *name,StringBuffer &mname,unsigned &num,unsigned &max,bool &replicate)
+static void parseFileName(const char *name,StringBuffer &mname,unsigned &num,unsigned &max,unsigned &stripeNum,unsigned &dirPerPart,bool &replicate)
 {
     // takes filename and creates mask filename with $P$ extension
+    const char *filename = name;
     StringBuffer nonrepdir;
-    replicate = setReplicateDir(name,nonrepdir,false);
-    if (replicate) 
-        name = nonrepdir.str();
+    if (!isContainerized())
+    {
+        replicate = setReplicateDir(name,nonrepdir,false);
+        if (replicate)
+            name = nonrepdir.str();
+    }
+    else
+        replicate = false; // Replicate dir is not supported in containerized environment
+
+    Owned<IPropertyTreeIterator> planesIter = getPlanesIterator("data", nullptr);
```

Comment:
This code should be revisited after this PR is merged. I've opened and linked : https://hpccsystems.atlassian.net/browse/HPCC-33151